From 9856330f54c181e388df6f261c705e01500cf0bb Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 6 Feb 2016 11:03:10 -0800
Subject: [PATCH 02/16] musl inspired fixed

stderr and stdout are not FILE pointers as per standards
define PAGE_MASK if its not defined already

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 host_applications/linux/libs/debug_sym/debug_sym.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/host_applications/linux/libs/debug_sym/debug_sym.c b/host_applications/linux/libs/debug_sym/debug_sym.c
index 8886f99..c7ace37 100755
--- a/host_applications/linux/libs/debug_sym/debug_sym.c
+++ b/host_applications/linux/libs/debug_sym/debug_sym.c
@@ -67,6 +67,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 # else
 #  define PAGE_SIZE   4096
 # endif
+#endif
+#ifndef PAGE_MASK
 #define PAGE_MASK   (~(PAGE_SIZE - 1))
 #endif
 
-- 
2.7.0

